operator/internal/manifests/internal/alerts/prometheus-alerts.yaml

```
@@ -209,3 +209,14 @@ groups:
     for: 1m
     labels:
       severity: warning
+  - alert: LokistackComponentsNotReady
+    annotations:
+      description: |-
+        The LokiStack "{{ $labels.stack_name }}" in namespace "{{ $labels.stack_namespace }}" has components that are not ready.
+      summary: "One or more LokiStack components are not ready."
+      runbook_url: "[[ .RunbookURL ]]#Lokistack-Pending-Components"
+    expr: |
+      sum  (lokistack_status_condition{reason="ReadyComponents", status="false"}) by (stack_name, stack_namespace, reason, status) == 1
```

- Made it resemble the style of the other query expressions
- Use `> 0` instead of `== 1` to be more failure tolerant (operator restarts, etc)
- Removed unused labels from aggregation


```suggestion
      sum (
        lokistack_status_condition{reason="ReadyComponents", status="false"}
      ) by (stack_name, stack_namespace)
      > 0
```